### PR TITLE
Feature/error toast on invalid valueoption selection 666

### PR DESCRIPTION
**Pull request by rinku-kumar-psl**
### Description
[Describe what this change achieves]

The below features are implemented in these changes:

- Default axes selected on Data config value options
- Error toast If the user selected any invalid or empty axis
- Error toast if the user trying to save the visualization, If f user selected any invalid or empty axis

Added these features for existing as well as new charts including Gauge, Histogram, and Treemap.

### Issues Resolved
[List any issues this PR will resolve]
#666 

### Check List
- [x]  New functionality includes an Error toast message on invalid value options selection while Apply and Save click, snapshot test cases, cypress test cases.
  - [ ] All tests pass, including unit test, and integration test, except a few test cases are failing at my end which are not related to this functionality.
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by rinku-kumar-psl** [comment]
Adding @spattnaik @abasatwar for reference.

File: dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.tsx [hunk 1]
```diff
@@ -95,8 +109,34 @@ export const ConfigPanel = ({ visualizations, setCurVisId, callback }: any) => {
     []
   );
 
+   // To check, If user empty any of the value options
+   const isValidValueOptionConfigSelected = useMemo(() => {
+    const valueOptions = vizConfigs.dataConfig?.valueOptions;
+    const { Bar, Line, Histogram, Pie, TreeMap, Gauge, HeatMap } = visChartTypes;
+    const isValidValueOptionsXYAxes = [Bar, Line, Histogram, Pie].includes(curVisId) &&
```
Comment (mengweieric) [inline dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.tsx:116]: Maybe move to the constants file, and name it to something like VIZ_CONTAIN_XY_AXIS? Just thinking other code pieces may also refer to this list, and visualizations that have x/y axes would be added and modified.

**Review by mengweieric — APPROVED** [review]

File: dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.tsx (hunk 1 above)
Comment (rinku-kumar-psl) [reply dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.tsx]: Thanks, @mengweieric 

As you suggested we have added this list in the constant file with the name VIZ_CONTAIN_XY_AXIS.

**Review by mengweieric — DISMISSED** [review]

**Review by mengweieric — APPROVED** [review]

**Review by eugenesk24 — APPROVED** [review]
Good!

**Pull request merged**